libqtile/layout/xmonad.py

```diff
@@ -1024,6 +1042,19 @@ def swap_main(self):
         elif self.align == self._down:
             self.swap_left()
 
+    @expose_command()
+    def grow_up(self):
+        """Grow/shrink main pane by moving it up
+
+        Logic corresponds to `MonadTall.grow_left()`
+        """
+        self.grow_left()
```

Comment:
Does the logic correspond? Aren't the values of `self.align` different in `MonadWide`?

Reply:
The logic works, yes; I've tested while changing alignment and the behavior holds.

MonadTall:

```
_left = 0
_right = 1
```

MonadWide:

```
_up = 0
_down = 1
```

That is how the two classes define their alignment so they will correspond to each other.

I could rewrite the functions in `MonadWide` using it's own alignment variables for clarity if that's preferable but it would be functionally identical.